### ISSUE TITLE
App crash on android

### ISSUE THREAD

**Issue by DamboRam** [freshplanet/ANE-Facebook#205]
java.lang.RuntimeException: Unable to resume activity {air.PromoscopeApp/com.facebook.LoginActivity}: java.lang.NullPointerException

Please help , or any advice

**Comment by wastedabuser** [comment]
For Android try this one: https://github.com/wastedabuser/ANE-Facebook
It should have these exceptions fixed!

**Comment by phpandroid** [comment]
Thank wastedabuser, you save my life :dancer:

**Comment by flashysquirrel** [comment]
and mine too !!!

**Comment by UmberFunk** [comment]
Thanks @wastedabuser. It's working but I spotted one issue. After pressing cancel on log in screen, it returns error saying that I am trying to open already opened session. When pressing connect second time, the log-in dialog is opened.

Are you going to add IOS support to your version of ANE? Or do you know any other version supporting both platforms and without crashing at log-in dialog?

**Comment by dr-andy** [comment]
For me both crash on Android when loggin in.

**Comment by UmberFunk** [comment]
@dr-andy It's great that Freshplanet provided opensouce ANE for FB and thank's for that, but I'd recommend you to switch to commercial ANE (Milkman, Distriqt, myflashlab). I had struggled a lot with this ANE. Checked a lot of versions and spent a lot of time making them work. After I managed to make any of them work, I was finding all kinds of bugs in them. Finally I managed to find best version of this ANE, made it work and build manager class for it. But, when I integrated it into our game, couldn't make it work anymore. Switching to commercial ANE took just several hours and I'm glad we did it.
